### PR TITLE
Configurable Discord Rich Presence icons and some localization

### LINKED ISSUE #3005: There is no possibility to configure Discord Rich Presence
I mean https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Client/Utility/DiscordRichPresence.cs file. 

Everything in it is hardcoded, cannot be localized and editable. 

image

in addition, for some reason, a big picture means the engine, and not the game itself

### PR DESCRIPTION
**Pull request by KIBORG04**
Now you can specify your icons if wizards den its want!!! (brought to you by ThetaGang!!!)

resolve https://github.com/space-wizards/RobustToolbox/issues/3005

### CODE REVIEW

**Comment by PJB3005** [comment]
Wouldn't this basically make it impossible to ever add better integration with Discord, for stuff like "join game" buttons?

**Comment by LudwigVonChesterfield** [comment]
Well right now it's impossible to do anything in regards to Discord presence modification without forking the game engine.

**Comment by KIBORG04** [comment]
> Wouldn't this basically make it impossible to ever add better integration with Discord, for stuff like "join game" buttons?

I just put the id in the config so that other forks can do something of their own. Who doesn't want to, nothing will change for them.

**Comment by KIBORG04** [comment]
Removed app id from cvar

File: Robust.Shared/CVars.cs [hunk 1]
```diff
@@ -1188,6 +1188,12 @@ protected CVars()
         public static readonly CVarDef<bool> DiscordEnabled =
             CVarDef.Create("discord.enabled", true, CVar.CLIENTONLY);
 
+        public static readonly CVarDef<string> DiscordRichPresenceMainIconId =
+            CVarDef.Create("discord.rich_main_icon_id", "devstation", CVar.CLIENTONLY);
+
+        public static readonly CVarDef<string> DiscordRichPresenceSecondIconId =
+            CVarDef.Create("discord.rich_second_icon_id", "logo", CVar.CLIENTONLY);
```
Comment (KIBORG04) [inline Robust.Shared/CVars.cs:1195]: I think my code doesn't not work because `CLIENTONLY`, not `SERVER`

**Pull request merged**